### ISSUE TITLE
Tablenames must not be separated by periods

### ISSUE THREAD

**Issue by RyanQuey** [jbolda/gatsby-source-airtable#239]
This issue seems similar to [issue 2](https://github.com/kevzettler/gatsby-source-airtable/issues/2) and [this issue](https://github.com/gatsbyjs/gatsby/issues/4751) and was solved in [issue 5155 in gatsby main repo](https://github.com/gatsbyjs/gatsby/pull/5155). 

But the difference is our column names use periods instead. For me, my column name is like `Logo.File` so is in the graphql schema as `Logo_File`. Fields that are have no periods work great for us, but when periods are there it returns `null`, similar to the issue before 5155 was merged. 

I noticed that [pr 5155 seems to solve for spaces and hyphens](https://github.com/gatsbyjs/gatsby/pull/5155/files#diff-fecae93e85b8bc9d054e84fd5a6f6b8fR80-R81), but not periods.

Is there a way to solve or get around this?

**Comment by jbolda** [comment]
We added our own function to take care of this before it was fixed upstream. If this doesn't work now and hasn't in the past, I think we could consider this a bug (and not worry about breaking other users). I believe this would be the relevant snippet. Happy to accept a PR with the change and an example testing it (or modify an existing one slightly).

https://github.com/jbolda/gatsby-source-airtable/blob/master/gatsby-node.js#L373
